README.md

```diff
@@ -1,9 +1,62 @@
-# Welcome to the axe DevTools HTML API Examples Repo
+# Welcome to axe DevTools HTML API Examples Repo
 
-Here you will find working examples of all of Deque's browser accessibility testing APIs.
+The axe DevTools HTML API Examples Repo contains the working examples of Deque browser accessibility testing APIs.
+
+The examples are categorized as follows:
+- coding language
+- testing framework and 
+- browser automation technology
+
+> **_NOTES:_**
+>- This project is currently in `Draft` state. If you do not find any relevant example (for Deque supported languages), contact [Deque HelpDesk](mailto:helpdesk@deque.com).
+>- You need a valid license to access and use our APIs and example projects. For more information, check out [our website](https://deque.com).
+
+The supported languages are as follows:
+
+### C#/.NET
```

Comment:
This is an accessibility violation https://dequeuniversity.com/rules/axe/4.4/heading-order?application=RuleDescription

Reply:
Agreed. Skipped the higher heading levels to keep the heading text size to be appealing.

As a workaround, bolded the heading text.